zebra-chain/src/sapling/commitment.rs

```diff
@@ -234,4 +286,102 @@ mod tests {
             assert_eq!(result, test_vector.output_point);
         }
     }
+
+    #[test]
+    fn test_add() {
+        let identity = ValueCommitment(jubjub::AffinePoint::identity());
+
+        let g = ValueCommitment(jubjub::AffinePoint::from_raw_unchecked(
+            jubjub::Fq::from_raw([
+                0xe4b3_d35d_f1a7_adfe,
+                0xcaf5_5d1b_29bf_81af,
+                0x8b0f_03dd_d60a_8187,
+                0x62ed_cbb8_bf37_87c8,
+            ]),
+            jubjub::Fq::from_raw([
+                0x0000_0000_0000_000b,
+                0x0000_0000_0000_0000,
+                0x0000_0000_0000_0000,
+                0x0000_0000_0000_0000,
+            ]),
+        ));
```

Comment:
This is a known point on the curve, I just needed any known point, this happens to be the generator of the curve because it's hardcoded into `jubjub`.

Reply:
What about making the point a constant outside the individual tests but inside `mod tests {` so we can add a comment explaining this is a known point that it is also the generator ?